read-fonts/src/collections/int_set/mod.rs

```diff
@@ -98,6 +98,10 @@ impl InDomain {
     pub fn value(&self) -> u32 {
         self.0
     }
+
+    pub fn new(value: u32) -> Self {
```

Comment:
Can you remove this, InDomain is not meant to be constructable outside of IntSet (see my above comment for how to work around).